common/version.go

```diff
@@ -0,0 +1,53 @@
+package common
+
+import (
+	"fmt"
+	"regexp"
+	"runtime/debug"
+	"strings"
+)
+
+type AppVersionInfo struct {
+	Version   string
+	BuildDate string
```

Comment:
Could we also include de the Git commit? It is provided by GoReleaser.

Reply:
:+1: 

Are you OK if I change the output by:

```
Mercure.rocks Hub version 0.10.0, commit 2159374, build at 2020-04-30T20:11:23Z
https://github.com/dunglas/mercure/releases/tag/v0.10.0
```

Reply:
perfect